featuretools/primitives/utils.py

```diff
@@ -7,6 +7,11 @@
 import featuretools.primitives
 from featuretools.utils import is_string
 
+try:
```

Comment:
can you add a comment in the code to this block about why were doing this?

Reply:
Done.